### ISSUE TITLE
The Chase

### ISSUE THREAD

**Issue by bobRas** [NaNoGenMo/2016#78]
I want to try and get something done this year.
Idea:
- Mix of Simulation (low level) and Story Compiler (high level)
- Combining different modules in sequential order (e.g. each Act gets a rather different implementation), connected by the "Story Spine" : villains chasing the heroes. Tension increases the closer the villain gets and the less places the heroes can escape to, while each Act gets its own smaller tension (not quite clear how yet).
- Biggest inspirations: [Simulationist Fantasy Novel](https://github.com/mattfister/nanogenmo2015), [There is an Orange here](https://github.com/flexo/nanogenmo2015), [MARYSUE](https://github.com/catseye/MARYSUE)

Thoughts?

**Comment by superMDguy** [comment]
Seems like a good combination.  I think the hardest part will be the low level simulation.  Many novels built using story compilers end up sounding really template-y.  If you can add a lot of variety to those lower levels, it should be great.  I think the chase idea is good, as it makes it easier to have page-turning tension.

**Comment by pointyointment** [comment]
>Simulationist Fantasy Novel, There is an Orange here

These will also be good inspiration for [my entry](74). Thanks for linking to them!

**Comment by bobRas** [comment]
Various thoughts:
- simulations "fail" in the narrative context whenever there are little to no persistent variables/states. in other words, you can't have a narrative if things keep going in circles. **There is an Orange here** works better than the **Simulationist Fantasy Novel** (as a narrative), because the number of participants are reduced. While the Fantasy characters keep moving and might get hungry/exhausted over time, those things can be overcome, and we're back to normal. On the other hand, the fantasy characters have a path and goal, while the hunger games participants keep doing the same things over and over again, because all they need to do is survive. I don't want characters to move back and forth between locations, or keep replacing inventory items.
- I'm not sure how long a simulation can run before it becomes too predictable, or too much like a template. It depends on the system of course, but it can't be too chaotic either. I decided to set a hard limit on how long a simulation may run before changing tracks. 1 page is 200-250 words. With 250 pages, we get 50000-62500 words. A range of 200-250 words seems good enough for me. Unfortunately, having so many different gear shifts is a bit too ambitious for one guy in one month, so I'll have to reduce this a bit.
- Each simulation should be (at least slightly) different. We can't keep doing the same things over and over for 250 pages. For example, have the first quarter (63 pages) be about analyzing the situation, the second and third quarter (125 pages) be about actually running away, and in the last quarter, have the heroes and villains confront each other. There needs to be additional differentiation within each quarter.
- Each quarter should take place in a new location, bloating the word count with description. Additionally, if we generate the setting, this might help with variety between novels.
- I should have more than one track running "simultaneously." After one page of Track 1, we may switch to one page of Track 2, and so on. This might reduce the repetition. I don't want to completely unrelated stories running at the same time; but rather take a page off **There is an Orange here** and give these additional Tracks to characters within the story, though they may have only tangentially related issues. I will try this with two heroes, and see what I can do with it. 4 tracks might be optimal?

Thoughts?

**Comment by superMDguy** [comment]
I agree with a lot of what you said.  Switching up the order should really help things.  MARYSUE was, in my opinion, one of the best novels, but each time the story happened in the same order.  Maybe even implementing 'flashbacks' would be interesting.  I wonder how that'd work out.  Switching out the locations should also add interest.
